modules/karma/karma.js

```diff
@@ -1,150 +1,150 @@
 exports.match = function(event, commandPrefix) {
-	return event.body.endsWith('++') || event.body.endsWith('--') || event.body === commandPrefix + 'karma';
+    return event.body.endsWith('++') || event.body.endsWith('--') || event.body === commandPrefix + 'karma';
 };
 
 exports.parseKarmaChange = function(message) {
-	var karma = 0, name = "";
-	for (var i = message.length - 2; i !== -1; i--) {
-		switch(message[i]){
-			case '+': karma++; break;
-			case '-': karma--; break;
-			default: name = message.substr(0, i+1); i = 0; break;
-		}
-	}
-	return {name: name.trim().toProperCase(), karma: karma};
+    var karma = 0, name = "";
+    for (var i = message.length - 2; i !== -1; i--) {
+        switch(message[i]){
+            case '+': karma++; break;
+            case '-': karma--; break;
+            default: name = message.substr(0, i+1); i = 0; break;
+        }
+    }
+    return {name: name.trim().toProperCase(), karma: karma};
 };
 
 exports.checkPerson = function(karma, person) {
-	var ps = person.split(' '),
-		ks = karma.split(' '),
-		overlap = ps.filter(function(n) {
-		return ks.indexOf(n) != -1
-	});
-	return overlap.length != 0;
+    var ps = person.split(' '),
+        ks = karma.split(' '),
+        overlap = ps.filter(function(n) {
+        return ks.indexOf(n) != -1
+    });
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 8 space characters but found 4.](https://www.codacy.com/app/theprojectshop/Kassy/file/2588679423/issues/source?bid=3282987&fileBranchId=3291254#l22)